csrankings-y.csv

```diff
@@ -278,6 +278,7 @@ Yin Tat Lee,University of Washington,http://yintat.com,nwHfwCIAAAAJ
 Yin Zhang 0001,University of Texas at Austin,http://www.cs.utexas.edu/~yzhang,DSEH8XsAAAAJ
 Yinan Jing,Fudan University,http://www.cs.fudan.edu.cn/?page_id=1848,NOSCHOLARPAGE
 Ying Cai,Iowa State University,http://www.cs.iastate.edu/~yingcai,N3Jc1zMAAAAJ
+Ying Chu,Shenzhen University,http://csse.szu.edu.cn/en/people?14717,pEZEgSkAAAAJ

```

Comment:
Web page does not work. Closing for now under the assumption that this person, like so many recently from Shenzhen, is in fact a postdoc and thus not eligible.